packages/sds/src/sds.ts

```diff
@@ -95,6 +118,36 @@ export class MessageChannel {
     }
   }
 
+  /**
+   * Sends a short-lived message without synchronization or reliability requirements.
+   *
+   * Sends a message without a timestamp, causal history, or bloom filter.
+   * Ephemeral messages are not added to the outgoing buffer.
+   * Upon reception, ephemeral messages are delivered immediately without
+   * checking for causal dependencies or including in the local log.
+   *
+   * See https://rfc.vac.dev/vac/raw/sds/#ephemeral-messages
+   *
+   * @param payload - The payload to send.
+   * @param callback - A callback function that returns a boolean indicating whether the message was sent successfully.
+   */
+  public sendEphemeralMessage(
```

Comment:
generally curious why you separate `ephemeral` from regular messages and not doing single `send` entry?

Reply:
Initially I tried that but decided that:
1. the logic is different enough that it doesn't look clean in the code
2. the concept is different enough that I want it to be as explicit as possible for the library consumer